notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Do you think that if we wrap here the one-hot encoding in a function, a user can reuse that function? or is it too specific?</p><p><br></p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
@miguelgfierro hmm.. somehow ReviewNB inserts an extra line between lines in the comments and doesn't support Markdown. We still can use ReviewNB as a complementary tool though. Super easy to read.

Reply:
> ReviewNB inserts an extra line between lines in the comments
I don't understand this very well. Are you viewing this on github or reviewnb?

Reply:
@miguelgfierro I left the above reply from ReviewNB and as you can see, each line has an extra line breaks there... (this comment is writing from github)